snaps/features/cron-jobs.md

```diff
@@ -64,6 +72,39 @@ export const onCronjob: OnCronjobHandler = async ({ request }) => {
 };
 ```
 
+</TabItem>
+<TabItem value="JSX" flaskOnly>
```

Comment:
JSX is not Flask-only. It will be in stable and Flask in v12. I also think we should make the JSX tab the default, and add a deprecation message to the functions tab.

Reply:
on it

Reply:
<img width="955" alt="image" src="https://github.com/MetaMask/metamask-docs/assets/3943143/1895101e-81fe-43c8-b0de-c43525fc9851">